core/domain/classifier_services.py

```diff
@@ -124,6 +124,35 @@ def classify_string_classifier_rule(state, normalized_answer):
 
     return None
 
+def check_classification_condition(state):
```

Comment:
It's an issue for the reviewer, if nothing else. Can you write, in this comment or a gist, one of the places where this function will be used, so that I can see how it's going to work in context? Then I can give you a recommendation on naming/placement.